central/src/main/java/org/glowroot/central/repo/ActiveAgentDao.java

```diff
@@ -121,9 +118,9 @@
         long revisedTo = CaptureTimes.getRollup(to, rollupIntervalMillis);
 
         Set<String> topLevelIds = new HashSet<>();
-        BoundStatement boundStatement = readTopLevelPS.get(rollupLevel).bind();
-        boundStatement.setTimestamp(0, new Date(from));
-        boundStatement.setTimestamp(1, new Date(revisedTo));
+        BoundStatement boundStatement = readTopLevelPS.get(rollupLevel).bind()
+            .setInstant(0, new Date(from).toInstant())
```

Comment:
```suggestion
            .setInstant(0, Instant.ofEpochMilli(from))
```

Reply:
Sure ! that's sound more than legit. 
Done